core/modules/menu/tests/menu.test

```diff
@@ -685,35 +687,46 @@ class MenuNodeTestCase extends BackdropWebTestCase {
     );
     $this->backdropPost('node/add/page', $edit, t('Save'));
     $node = $this->backdropGetNodeByTitle($node_title);
-    // Assert that there is no link for the node.
-    $this->backdropGet('user');
+    // Assert that no link has been created for the node in the list of links
+    // for the "Primary navigation" menu.
+    $this->backdropGet('admin/structure/menu/manage/main-menu');
```

Comment:
Not sure why we were checking in the `/user` path before 🤷 ...anyway, the block for the "Primary navigation" menu may not be rendered for some reason; better check directly in the list of links for that menu.